pkg/deps/maven_test.go

```diff
@@ -0,0 +1,56 @@
+package deps_test
```

Comment:
About this test case, I can't see you check the expected jars list. Could you enhance this test case? And notice, `test` and `provided` scopes should be checked, both these are not dependency check scope.

Reply:
> And notice, `test` and `provided` scopes should be checked, both these are not dependency check scope.

This is misleading, they should NOT be checked. I edited your comment

Reply:
If you check the method this PR does (using `mvn copy-dependencies`), it only copies runtime dependencies, the same way we did in main repo to check licenses dependencies.

Reply:
I noticed this use `mvn copy-dependencies`, I mean the test case should verify.

Reply:
add option `-DincludeScope=runtime` to `mvn copy-dependencies` for excluding `test`, `provided` and `system`, and modify the test case